package.json

```diff
@@ -21,6 +21,7 @@
     "@babel/core": "^7.11.6",
     "@babel/preset-env": "^7.11.5",
     "@babel/preset-react": "^7.10.4",
+    "@babel/runtime": "^7.12.5",
```

Comment:
Do we really need to use this package? Can we remove it?
npm uninstall @babel/runtime